fedimint-api-client/src/api/iroh.rs

```diff
@@ -58,9 +58,14 @@ impl IrohConnector {
         peers: BTreeMap<PeerId, SafeUrl>,
         iroh_dns: Option<SafeUrl>,
```

Comment:
I tried to make this an `Option<Vec<_>>` too, but it spiraled a bit out of control. If you feel strongly about it I can add it as a separate commit @dpc.